Source/SwiftLintFramework/Rules/DuplicateImportsRule.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  DuplicateImportsRule.swift
+//  SwiftLint
+//
+//  Created by Samuel Susla on 1/13/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+extension Line {
+    var importIdentifier: Substring? {
+        return self.content.split(separator: " ").last
+    }
+
+    func areImportsDuplicated(with otherLine: Line) -> Bool {
+        guard let firstImportIdentifiers = self.importIdentifier?.split(separator: "."),
+              let secondImportIdentifiers = otherLine.importIdentifier?.split(separator: ".")
+            else { return false }
+
+        return zip(firstImportIdentifiers, secondImportIdentifiers).reduce(true) { result, sequences in
+            return result && sequences.0 == sequences.1
+        }
+    }
+}
+
+public struct DuplicateImportsRule: ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "duplicate_imports",
+        name: "Duplicate Imports",
+        description: "Imports should be unique.",
+        kind: .style,
+        nonTriggeringExamples: [
+            "import A\nimport B\nimport C",
+            "import A.B\nimport import A.C"
```

Comment:
There's a double `import` for `A.C`.

Reply:
@ornithocoder thank you, I've fixed that.